src/Code.js

```diff
@@ -89,6 +99,44 @@ function action(req) {
     return ContentService.createTextOutput(JSON.stringify(response)).setMimeType(ContentService.MimeType.JSON);
   }
 
+  if (captcha) {
+    switch (captcha.type) {
+      case 'recaptcha_v2':
+        const siteKey = jsonData['g-recaptcha-response'];
+
+        if (!siteKey) {
+          response = {
+            status: 'error',
+            message: 'reCAPTCHA verification under key \'g-recaptcha-response\' is required.',
+          };
+          return ContentService.createTextOutput(JSON.stringify(response)).setMimeType(ContentService.MimeType.JSON);
+        }
+
+        var captchaResponse = UrlFetchApp.fetch('https://www.google.com/recaptcha/api/siteverify', {
```

Comment:
Maybe you can use `const` as it is used every where.

Reply:
Hoopsie, copy&paste from docs. Nice catch.